concepts/booleans/about.md

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Don't forget to revert all changes that are not tuple related, see https://github.com/exercism/cairo/pull/106#discussion_r1700266710